error.go

```diff
@@ -24,3 +24,10 @@ var invalidConfigError = microerror.New("invalid config")
 func IsInvalidConfig(err error) bool {
 	return microerror.Cause(err) == invalidConfigError
 }
+
+var itemsNotFoundError = microerror.New("items not found")
```

Comment:
Why itemsNotFoundError instead of notFoundError like everywhere else?

Reply:
No strong reason.